src/core/dfn.js

```diff
@@ -1,12 +1,28 @@
 // @ts-check
 // Module core/dfn
 // - Finds all <dfn> elements and populates conf.definitionMap to identify them.
+// - Also populates abbrMap from core/inlines for definitions that have an abbreviation
 
 import { getDfnTitles } from "./utils";
 import { registerDefinition } from "./dfn-map";
 
 export const name = "core/dfn";
 
+export function addDfnAbbrtoMap(abbrMap) {
+  document.querySelectorAll("dfn").forEach(dfn => {
```

Comment:
This seems backwards, because we are doing this query on line 27. 

How about create a new module called data-abbr.js instead that runs before inlines.js... that module only processes `querySelectorAll("dfn[data-abbr]")` and does the appropriate text replacements. It can own and export `abbrMap`, which inlines.js can then import.

Reply:
Thats exactly what I wanted to do in the first place. But was hesistant in adding a new module. Thanks!